### PR TITLE
openmw-tes3mp: 0.7.0-alpha -> 2019-06-09

### PR DESCRIPTION
**Pull request by ghost**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by gnidorah** [comment]
cc @abbradar

File: pkgs/games/openmw/tes3mp.nix [hunk 1]
```diff
@@ -1,102 +1,72 @@
...
+  rakNet = fetchFromGitHub {
     owner = "TES3MP";
     repo = "CrabNet";
-    rev = "ab1306050fe0f5b0f9c4f56893a79e56a9459567";
-    sha256 = "03q76pjv9mdi7w832b23q1mj4r2wb0hsnh4kpvwai607g04l0pp0";
+    rev = "4eeeaad2f6c11aeb82070df35169694b4fb7b04b";
+    sha256 = "0p0li9l1i5lcliswm5w9jql0zff9i6fwhiq0bl130m4i7vpr4cr3";
   };
-  CoreScripts = fetchFromGitHub {
+  rakNetLibrary = (stdenv.mkDerivation {
```
Comment (abbradar) [inline pkgs/games/openmw/tes3mp.nix]: Parentheses are unneeded here.

File: pkgs/games/openmw/tes3mp.nix [hunk 2]
```diff
@@ -1,102 +1,72 @@
...
-    sha256 = "012f50f9jd29qcdww2vk4habg6pmxvxl0q6rrjq8xchb0566712q";
-  };
-  CallFF = fetchFromGitHub {
-    owner = "Koncord";
-    repo = "CallFF";
-    rev = "da94b59ffe95d45bf98b9264e3d1279c9f6ebb6b";
-    sha256 = "10wgiqmknh0av968c6r74n5n2izxsx8qawfrab57kkmj9h0zp0pm";
-  };
-  CrabNet = fetchFromGitHub {
+  compatHash = "292536439eeda58becdb7e441fe2e61ebb74529e";
```
Comment (abbradar) [inline pkgs/games/openmw/tes3mp.nix]: Let's mention specifically `TES3MP_STABLE_VERSION_FILE` here in comments for clarity.

File: pkgs/games/openmw/tes3mp.nix [hunk 3]
```diff
@@ -1,102 +1,72 @@
...
+  rakNet = fetchFromGitHub {
     owner = "TES3MP";
     repo = "CrabNet";
-    rev = "ab1306050fe0f5b0f9c4f56893a79e56a9459567";
-    sha256 = "03q76pjv9mdi7w832b23q1mj4r2wb0hsnh4kpvwai607g04l0pp0";
+    rev = "4eeeaad2f6c11aeb82070df35169694b4fb7b04b";
```
Comment (abbradar) [inline pkgs/games/openmw/tes3mp.nix]: Let's add a comment saying this revision is just latest master, to ease update.

File: pkgs/games/openmw/tes3mp.nix [hunk 4]
```diff
@@ -1,102 +1,72 @@
...
+    installPhase = ''
+      install -Dm755 lib/libRakNetLibStatic.a $out/lib/libRakNetLibStatic.a
+    '';
+  });
+  coreScripts = fetchFromGitHub {
     owner = "TES3MP";
     repo = "CoreScripts";
-    rev = "1e9f69f98051b2639b18203f989ffbd0a4b427ea";
-    sha256 = "03ysi7rh0k78kv4slvmkxpymxvdpr8b6hwr1lvjdgq7rq0ljy0lg";
+    rev = "71e15fa3b1d5131b6607ba1589f41c06672ce376";
```
Comment (abbradar) [inline pkgs/games/openmw/tes3mp.nix]: Same, let's leave a comment there.

File: pkgs/games/openmw/tes3mp.nix [hunk 5]
```diff
@@ -1,102 +1,72 @@
...
+  name = "openmw-tes3mp-${version}";
 
   src = fetchFromGitHub {
-    owner = "GrimKriegor";
-    repo = "TES3MP-deploy";
-    rev = "1dd78a3e2cf9f4fe85bf7ca9c393251968a9c325";
-    sha256 = "1bp9c4kds9q0xhbn4sxb7n0f6rvb45gzx7ljdgc56wz4j5rfi3xn";
+    owner = "TES3MP";
+    repo = "openmw-tes3mp";
+    rev = "01804af100785bc2c162d568258d9662012627a3";
```
Comment (abbradar) [inline pkgs/games/openmw/tes3mp.nix]: Comment that this one is latest in branch 0.7.0.

**Review by abbradar — COMMENTED** [review]
Let's document update process more; other than that, nice work!

**Comment by gnidorah** [comment]
@abbradar
> Let's document update process more; other than that, nice work!

Basically I've stopped to build with tes3mp-deploy.sh and that made expression simpler. Also installation is now made by official cmake scripts

File: pkgs/games/openmw/tes3mp.nix (hunk 3 above)
Comment (gnidorah) [reply pkgs/games/openmw/tes3mp.nix]: @abbradar Actually its fixed in script in counter to revisions of other components https://github.com/GrimKriegor/TES3MP-deploy/blob/master/tes3mp-deploy.sh#L589

Comment (abbradar) [reply pkgs/games/openmw/tes3mp.nix]: I mentioned it, thanks!

**Comment by gnidorah** [comment]
@abbradar Thank you!

**Pull request closed without merging**